### DISCUSSION TITLE
Operator "Métropole Nice Côte d'Azur / Veloway by Transdev" instead of "Véolia"

### DISCUSSION THREAD

**Discussion by V75014** [osmlab/name-suggestion-index#6433] [Q&A]
Hello,

I am a new contributor for GitHub.

I don't know how to change operator for Vélo Bleu in Nice (France)

https://github.com/osmlab/name-suggestion-index/blob/b9e9602f90e99fcb2d86191d71dc785d56d14e68/data/brands/amenity/bicycle_rental.json#L3677

The new operator is "Métropole Nice Côte d'Azur / Veloway by Transdev".

You can see https://www.velobleu.org/mentions-legales/

Thanks
Vincent / V75014

**Comment by bhousel** [comment]
Hi @V75014 and welcome 👋 
These properties like `operator` are "tags" that end up applied to the features in OpenStreetMap - so we normally try to have them match the values that are already in use by OpenStreetMap.   

OpenStreetMap contributors do choose all kinds of values for the `operator` tag, and we collect them in this other project:
https://raw.githubusercontent.com/ideditor/nsi-collector/main/dist/osm/operators_all.json

Sometimes it really does contain a legal name, and sometimes it just contains a short phrase that the local community has agreed on.

So looking through this file - it does seem that "Véolia" is the accepted value, and a change like "Métropole Nice Côte d'Azur / Veloway by Transdev" would not be welcomed.

The good news - it doesn't _really_ matter much what the `operator` is, once the `operator:wikidata` tag has been set up and we can use that as the identifying value.
